sdk/tests/utilities/test_token_refresh.py

```diff
@@ -51,9 +52,7 @@ def test_get_token_with_proxy(self):
         if secrets["proxy"].get("address", None) is None:
             self.skipTest(f"missing proxy configuration")
 
-        secrets_file = TempFileManager.create_temp_file(secrets)
-
-        original_token, refresh_token = tu.get_okta_tokens(secrets_file.name)
```

Comment:
Don't need to get these tokens anymore as the RefreshingToken doesn't need an initial access token or initial refresh token as it can get them itself.